### PR TITLE
fix active submenu item via project.yml on landing

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: hugo/layouts/partials/header.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {{ $ctx := .ctx }}
 {{ $categories := $ctx.Site.Data.categories.categories.contents }}
 {{ $project := $ctx.Site.Data.project }}
-{{ $category := default "" $ctx.Site.Data.project.category }}
+{{ $isDocSite := default false $project.is_doc_site }}
```
Comment (dpordomingo) [inline hugo/layouts/partials/header.html]: if `$ctx.Site.Data.project` is set, then the site is a documentation site, isn't it?

Comment (erizocosmico) [inline hugo/layouts/partials/header.html]: Nope, the landing has it for the intro page

**Comment by erizocosmico** [comment]
But how does adding a class to the body will make active some submenu?

**Comment by dpordomingo** [comment]
Not magically, I know ;)
It would be done from the css.

Usually an "active" button is not (should not be) enabled because it is already active; and in this case, it is not the same kind of "activeness" because it keeps being clickable -> that's because I think it makes sense to do the styling from the css, something like:
```
.docummentation-site .projectButton {
    @include activeButton;
}
```

**Comment by erizocosmico** [comment]
Should we leave this, then, until you fix it on your side?

**Pull request closed without merging**